app.js

```diff
@@ -0,0 +1,67 @@
+const bookSection = document.getElementById('book-section');
+
+function add(e) {
+  const name = document.getElementById('input-name').value;
+  const author = document.getElementById('input-author').value;
+
+  const book = {
+    name,
+    author,
+  };
+
+  if (localStorage.getItem('books') === null) {
+    const books = [];
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  } else {
+    const books = JSON.parse(localStorage.getItem('books'));
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+  document.getElementById('form').reset();
+  e.preventDefault();
+  window.location.reload();
```

Comment:
- [ ] [OPTIONAL] Good job on implementing the add function, but it is not advisable to use the `window.location.reload` method often as it slows the page down.